app/lambda/create_session/lambda_function.py

```diff
@@ -0,0 +1,65 @@
+import json
+import sys
+import time
+import uuid
+
+sys.path.append("../")
+import boto3
+from lambda_base import LambdaBase
+from constant_variables import *
+from models import Session
+
+
+class CreateSessionLambda(LambdaBase):
```

Comment:
It is a common practice to start classnames in caps and follow camel case/underscores with the initial letter not in caps for functions and variables.

In the datastore class , you can see example of this
Here are [examples](https://github.com/aws-samples/medical-transcription-analysis/pull/21/files#diff-808da8b40b7c6f4c10d19edaa01d41eeR12) for variables from Datastore class  -> camelcase

Here are [examples](https://github.com/aws-samples/medical-transcription-analysis/pull/21/files#diff-808da8b40b7c6f4c10d19edaa01d41eeR49) for functions from Datastore class -> underscore

I would suggest following a uniform pattern for all functions and all variables across the entire package for consistency

Reply:
all functions and variables changed to camelcase. https://github.com/aws-samples/medical-transcription-analysis/blob/691348b78d89ba8652fa177e9a002b4466b0ef6b/app/lambda/datastore.py#L49